src/makeEventList.js

```diff
@@ -250,10 +257,16 @@ function makeEventList(aimsTimeTable, customEventList) {
             break;
           }
 
+          if (courseName != null) {
```

Comment:
Can you move this up with the previous scope? 
It doesn't seem right in this loop.